test/plugin/scenarios/spring-tx-scenario/config/expectedData.yaml

```diff
@@ -0,0 +1,168 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+registryItems:
+  applications:
+    - {spring-tx-scenario: 2}
+  instances:
+    - {spring-tx-scenario: 1}
+  operationNames:
+    - spring-tx-scenario: [Mysql/JDBI/Statement/execute, Mysql/JDBI/Connection/commit,
+                           Mysql/JDBI/PreparedStatement/executeUpdate, /case/spring-tx-case, /case/healthCheck,
+                           Mysql/JDBI/Connection/close, Mysql/JDBI/Statement/executeQuery]
+  heartbeat: []
+segmentItems:
+  - applicationCode: spring-tx-scenario
+    segmentSize: nq 0
+    segments:
+      - segmentId: not null
+        spans:
+          - operationName: TX/get/test.org.apache.skywalking.apm.testcase.spring.transaction.service.impl.DemoServiceImpl.doBiz
```

Comment:
I am not familiar with this.  I think `operationName` is a little long. Maybe we can use `t.o.s.a.t.s.t.s.i.DemoServiceImpl.doBiz`.

Reply:
If we want to support this, please be an optional config.

Reply:
Good proposal. I'll finish it.

Reply:
Hi, store method name in tags? FYI

Reply:
Don't save duplicated inform. Otherwise, there is no point to implement short name. Use config if you want to give user options. But, you need make sure converting to short is quick enough.

Reply:
```yaml
operationName: TX/get
tags: 
 - {key: "method", value:"test.org.apache.skywalking.apm.testcase.spring.transaction.service.impl.DemoServiceImpl.doBiz"}
```
My idea should be like this,  can ignore.

Reply:
I just rechecked the specification and I am also wondering whether this operation name is **too specific**, as suggested in [the specification](https://github.com/opentracing/specification/blob/master/specification.md#start-a-new-span)

Reply:
We don't need to follow the OpenTracing, as it is not a specific at all. I would like to discuss what kind of operation name makes more sense to user when they see our trace view. Could we discuss based on this?